plugin/src/main/java/com/gtnewhorizons/retrofuturagradle/mcp/MCPTasks.java

```diff
@@ -985,21 +990,24 @@ private void afterEvaluate() {
                 });
             }
         }
+
         if (mcExt.getUsesFml().get()) {
             deobfuscationATs.builtBy(taskExtractForgeUserdev);
             if (mcMinor <= 8) {
                 deobfuscationATs.from(userdevFile(Constants.PATH_USERDEV_FML_ACCESS_TRANFORMER));
             }
 
-            taskPatchDecompiledJar.configure(task -> {
-                task.getPatches().builtBy(taskExtractForgeUserdev);
-                task.getInjectionDirectories().builtBy(taskExtractForgeUserdev);
-                if (mcMinor <= 8) {
-                    task.getPatches().from(userdevFile("fmlpatches.zip"));
-                }
-                task.getInjectionDirectories().from(userdevDir("src/main/java"));
-                task.getInjectionDirectories().from(userdevDir("src/main/resources"));
-            });
+            if (!"1.12.2".equals(mcExt.getMcVersion().get()) || mcExt.getUsesForge().get()) {
```

Comment:
Use `mcMinor <= 8` instead of checking for "not 1.12.2" (7-8 are fg1.2, 9-12 are fg2.3)